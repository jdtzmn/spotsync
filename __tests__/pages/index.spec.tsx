import * as React from 'react'
import { shallow } from 'enzyme'
import Index from '../../pages/index'

describe('Index', () => {
  let wrapper
  beforeEach(() => {
    wrapper = shallow(<Index />)
  })

  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
