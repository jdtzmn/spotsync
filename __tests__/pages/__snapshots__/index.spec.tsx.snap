// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index should render correctly 1`] = `
<main>
  <Navbar />
  <div>
    <div>
      <Fade
        delay={150}
      >
        <img
          alt=""
          src=""
        />
        <img
          alt=""
          src=""
        />
      </Fade>
      <div>
        <div>
          <Fade
            bottom={true}
          >
            <h1>
              Spotify in sync
            </h1>
          </Fade>
          <Fade
            bottom={true}
            delay={200}
          >
            <sub>
              Spotsync makes multi-device listening a breeze
            </sub>
          </Fade>
        </div>
        <div>
          <Fade
            bottom={true}
            delay={300}
          >
            <LinkRoutes
              route="/auth/login"
            >
              <button
                data-cypress="get-started"
              >
                Get Started
              </button>
            </LinkRoutes>
          </Fade>
          <Fade
            bottom={true}
            delay={350}
          >
            <sub>
              By logging in with Spotify
            </sub>
          </Fade>
        </div>
      </div>
    </div>
    <div>
      <Fade
        bottom={true}
      >
        <img
          alt=""
          src=""
        />
      </Fade>
      <div>
        <Fade
          bottom={true}
          delay={400}
        >
          <h1>
            Get the Party Started
          </h1>
        </Fade>
        <Fade
          bottom={true}
          delay={600}
        >
          <sub>
            In three simple steps
          </sub>
        </Fade>
        <div
          key="0"
        >
          <Fade
            bottom={true}
            delay={400}
          >
            <h2>
              1
            </h2>
            <h2>
              Create a room
            </h2>
            <h3>
              A personalized room just for you
            </h3>
          </Fade>
        </div>
        <div
          key="1"
        >
          <Fade
            bottom={true}
            delay={400}
          >
            <h2>
              2
            </h2>
            <h2>
              Share the code
            </h2>
            <h3>
              Invite people to join your room
            </h3>
          </Fade>
        </div>
        <div
          key="2"
        >
          <Fade
            bottom={true}
            delay={400}
          >
            <h2>
              3
            </h2>
            <h2>
              Add to queue
            </h2>
            <h3>
              It's as easy as that
            </h3>
          </Fade>
        </div>
      </div>
    </div>
    <div>
      <Fade
        bottom={true}
      >
        <img
          alt=""
          src=""
        />
      </Fade>
      <Fade
        bottom={true}
        delay={400}
      >
        <h1>
          Every person can add songs to the queue
        </h1>
      </Fade>
      <Fade
        bottom={true}
        delay={400}
      >
        <sub>
          The songs will keep on playing
        </sub>
      </Fade>
    </div>
    <div>
      <Fade
        bottom={true}
        delay={200}
      >
        <h1>
          Ready to Go?
        </h1>
      </Fade>
      <Fade
        bottom={true}
        delay={300}
      >
        <LinkRoutes
          route="/auth/login"
        >
          <button
            data-cypress="start-a-party"
          >
            Start a Party
          </button>
        </LinkRoutes>
      </Fade>
      <Fade
        bottom={true}
        delay={350}
      >
        <Tada
          delay={1000}
        >
          <sub>
            (It's free)
          </sub>
        </Tada>
      </Fade>
    </div>
    <Footer />
  </div>
</main>
`;
